CHANGELOG.md

```diff
@@ -3,6 +3,63 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.0
+
+_Sep 1, 2022_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```

Comment:
```suggestion
- 🎁 Implement Lazy loading (#5214) @DanailH

  Pro users now can try experimental `lazyLoading` feature.
  Implement lazy infinite skeleton loading with a few settings.

  To enable this feature, add `experimentalFeatures={{ lazyLoading: true }}`.
  Lazy loading feature has a few other prerequisites.
  See the [documentation](https://mui.com/x/react-data-grid/row-updates/#lazy-loading) to explore example in detail.

- 🚀 Improve `pickers` focus management (#5820) @alexfauquette
- 🎉 Enable disabling `day` on date range picker depending on `position` (#5773) @alexfauquette
- ✨ Various `DataGrid` and `pickers` improvements
- 📚 Documentation improvements
- 🐞 Bugfixes
```

Reply:
Thanks for adding the highlights. I was about to look into them